### ISSUE TITLE
MAPE should be multiplied by 100

### ISSUE THREAD

**Issue by TKassis** [timeseriesAI/tsai#152]
I noticed that the Mean Absolute Percentage Error (MAPE) is calculated as a fraction and not percent.

```python
def mape(inp,targ):
    "Mean absolute percentage error between `inp` and `targ`."
    inp,targ = flatten_check(inp, targ)
    return (torch.abs(inp - targ) / torch.clamp_min(targ, 1e-8)).mean()
```

I propose:
```python
def mape(inp,targ):
    "Mean absolute percentage error between `inp` and `targ`."
    inp,targ = flatten_check(inp, targ)
    return (torch.abs(inp - targ) / torch.clamp_min(targ, 1e-8)).mean() * 100
```

**Comment by oguiza** [comment]
Sorry, but this is aligned to the rest of the metrics in fastai (like accuracy, for example). I prefer to leave it this way to avoid any confusion.

**Comment by TKassis** [comment]
Ah I see, I wonder though if we can indicate that somewhere in the doc string or docs. It was really confusing to me for several days until I looked at the source code. The MAPE metric was not agreeing with my calculated MAPE. I think given the 'Percentage' in the name people might expect it to be percent and not a fraction.
